documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,287 @@
+
+
+.Deleting a migration plan does not remove temporary resources
+
+Deleting a migration plan does not remove temporary resources.
+
+* Importer pods
+* Conversion pods
+* Config maps
+* Secrets
+* Failed VMs
+* Data volumes.
+
+To clean up the the temporary resources, you must archive a migration plan before deleting it. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
+
+
+.Unclear error status message for VM with no operating system
+
+The error status message for a VM with no operating system on the *Plans* page of the web console does not describe the reason for the failure. link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[(BZ#22008846)]
+
+
+.Migration of virtual machines with encrypted partitions fails during a conversion (vSphere only)
+
+vSphere only: Migrations from {rhv-short} and {osp} do not fail, but the encryption key might be missing on the target {ocp} cluster.
+
+
+.Migration fails during precopy/cutover while performing a snapshot operation on the source VM
+
+Warm migration from {rhv-short} fails if a snapshot operation is triggered and running on the source VM at the same time as the migration is scheduled. The migration does not wait for the snapshot operation to finish. link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
+
+
+.Unable to schedule migrated VM with multiple disks to more than one storage class of type `hostPath`
+
+When migrating a VM with multiple disks to more than one storage classes of type `hostPath`, it might happen that a VM cannot be scheduled. Workaround: Use shared storage on the target {ocp} cluster.
+
+
+.Warm migrations do not support guest operating systems
+
+Warm migrations and migrations to remote {ocp} clusters from vSphere do not support the same guest operating systems that are supported in cold migrations and migrations to the local {ocp} cluster. RHEL 8 and RHEL 9 might cause this limitation.
+See link:https://access.redhat.com/articles/1351473[Converting virtual machines from other hypervisors to KVM with virt-v2v in RHEL 7, RHEL 8, and RHEL 9] for the list of supported guest operating systems.
+
+
+.VMs from vSphere with RHEL 9 guest operating system can start with network interfaces that are down
+
+When migrating VMs that are installed with RHEL 9 as guest operating system from vSphere, the network interfaces of the VMs could be disabled when they start in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-491[(MTV-491)]
+
+
+.Leftover `ovirtvolumepopulator` from failed migration causes the plan to stop in the `CopyDisks` phase
+
+An earlier failed migration can leave an outdated `ovirtvolumepopulator` in the project of a new plan for the same VM. The `CreateDataVolumes` phase does not create populator PVCs when transitioning to `CopyDisks`, causing the `CopyDisks` phase to stay indefinitely. link:https://issues.redhat.com/browse/MTV-929[(MTV-929)]
+
+
+.Unclear error message when Forklift fails to build or create a PVC
```

Comment:
this is also fixed

Reply:
move to Resolved Issues or delete, please